lib/sensors/include/EncoderAbsolute_PWM.h

```diff
@@ -1,8 +1,6 @@
 #pragma once
 
-#ifndef PWM_IN_DEFAULT_NUM_SAMPLES_TO_AVERAGE
```

Comment:
i dont think you need this change. this is not related to fie header guards. this just optionally declares a macro. not rly related to what you're doing.